WebHostLib/static/assets/tutorial/meritous/setup_en.md

```diff
@@ -0,0 +1,64 @@
+# Meritous Randomizer Setup Guide
+
+## Required Software
+
+[Meritous Gaiden](https://github.com/FelicitusNeko/meritous-ap/releases)
```

Comment:
```suggestion
Download the game from the [Meritous Gaiden GitHub releases page](https://github.com/FelicitusNeko/meritous-ap/releases)
```

Generally, I ask that hyperlinks are descriptive of the page they lead to. This is with intent of aiding those using screen readers.

Reply:
Done.